pwiz_tools/Skyline/Model/Databinding/Entities/TransitionResult.cs

```diff
@@ -85,6 +85,17 @@ public void ChangeChromInfo(EditDescription editDescription, Func<TransitionChro
         [Format(NullValue = TextUtil.EXCEL_NA)]
         public int? PointsAcrossPeak { get { return ChromInfo.PointsAcrossPeak; } }
 
+        [Format(Formats.RETENTION_TIME, NullValue = TextUtil.EXCEL_NA)]
+        public double? AverageCycleTime
+        {
+            get
+            {
+                return StartTime.HasValue && EndTime.HasValue && PointsAcrossPeak.HasValue

```

Comment:
You can simplify this by doing:
                return (EndTime - StartTime) * 60 / PointsAcrossPeak;

Reply:
That's useful, I didn't know that works with nullables.